packages/fxa-content-server/app/scripts/models/auth_brokers/oauth-redirect.js

```diff
@@ -107,7 +107,10 @@ export default BaseAuthenticationBroker.extend({
         this.relier.set('service', this.relier.get('clientId'));
         this._metrics.setService(this.relier.get('clientId'));
       }
-      this._metrics.logEvent('pairing.signin.success');
+
+      if (result.action === 'pairing') {
```

Comment:
The bug was here, we needed to check that this was indeed the pairing flow before logging complete.